mono/metadata/appdomain.h

```diff
@@ -82,10 +82,10 @@ mono_domain_get_id         (MonoDomain *domain);
 MONO_API const char *
 mono_domain_get_friendly_name (MonoDomain *domain);
 
-MONO_API mono_bool
+MONO_API MONO_RT_EXTERNAL_ONLY mono_bool
 mono_domain_set            (MonoDomain *domain, mono_bool force);
 
-MONO_API void
+MONO_API MONO_RT_EXTERNAL_ONLY void
```

Comment:
“Internal” is public? :(

Reply:
I mentioned this in the PR description but yeah, seems someone just made it public alongside everything else in the file in an older commit... It makes for really awkward naming, but its behavior isn't even the same as `mono_domain_set` and I'm trying to avoid API breakage.

Reply:
Avoid breakage definitely.
Stuff will get broken, but at least by trying not to, less is broken.